### DISCUSSION TITLE
Basic implementation of the flexbox layout is merged!

### DISCUSSION THREAD

**Discussion by tordex** [litehtml/litehtml#286] [Announcements]
Basic implementation of the flexbox layout is merged to the master branch. More then 400 [tests from W3C](http://test.csswg.org/suites/css-flexbox-1_dev/nightly-unstable/html/toc.htm) are passed.

**Comment by stasoid** [comment]
Cool stuff, thank you. 
Wasn't aware of these tests. Only knew https://web-platform-tests.org/ , but those require JS.
